lib/render/editor/editor.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+<head>
+  <!-- blockly -->
+  <link href="./gs-element-blockly.html" rel="import" />
+</head>
+
+<body>
+
+<dom-module id="blockly-container">
+  <template>
+    <gs-element-blockly id="blocklyElement"></gs-element-blockly>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'blockly-container',
+      ready: function() {
+        setTimeout(() => {
+          this.$.blocklyElement.workspace.addChangeListener(() => {
+            $("#custom-editor-value")[0].value = this.$.blocklyElement.generateCode();
+          });
+        });
+      }
+    });
+  </script>
+</dom-module>
+
+<blockly-container></blockly-container>
```

Comment:
I would delete this, since you're already inserting the `<blockly-container>` element by javascript